doc/api/child_process.markdown

```diff
@@ -230,20 +240,22 @@ const child = execFile('node', ['--version'], (error, stdout, stderr) => {
   * `execPath` {String} Executable used to create the child process
   * `execArgv` {Array} List of string arguments passed to the executable
     (Default: `process.execArgv`)
-  * `silent` {Boolean} If true, stdin, stdout, and stderr of the child will be
+  * `silent` {Boolean} If `true`, stdin, stdout, and stderr of the child will be
     piped to the parent, otherwise they will be inherited from the parent, see
     the `'pipe'` and `'inherit'` options for [`child_process.spawn()`][]'s
-    [`stdio`][] for more details (default is false)
-  * `uid` {Number} Sets the user identity of the process. (See setuid(2).)
-  * `gid` {Number} Sets the group identity of the process. (See setgid(2).)
+    [`stdio`][] for more details (Default: `false`)
+  * `uid` {Number} Sets the user identity of the process. (See
+    [`setuid(2)`][].)
+  * `gid` {Number} Sets the group identity of the process. (See 
+    [`setgid(2)`][].)
 * Return: {ChildProcess}
 
 The `child_process.fork()` method is a special case of
 [`child_process.spawn()`][] used specifically to spawn new Node.js processes.
-Like `child_process.spawn()`, a `ChildProcess` object is returned. The returned
-`ChildProcess` will have an additional communication channel built-in that
-allows messages to be passed back and forth between the parent and child. See
-[`ChildProcess#send()`][] for details.
+Like [`child_process.spawn()`][], a [`ChildProcess`][] object is returned. The
+returned [`ChildProcess`][] will have an additional communication channel built-in
+that allows messages to be passed back and forth between the parent and child.
+See [`child.send()`][] for details.
```

Comment:
This is stylistically inconsistent, sometimes you have child_process.spawn(), sometimes child.send(). I suggest uniformly using the shorter form throughout this page.